src/test/java/pl/tomaszgronski/projektksiegowy/invoice/InvoiceBookTestMock.java

```diff
@@ -0,0 +1,76 @@
+package pl.tomaszgronski.projektksiegowy.invoice;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import pl.tomaszgronski.projektksiegowy.invoice.db.Database;
+import pl.tomaszgronski.projektksiegowy.invoice.model.Invoice;
+import pl.tomaszgronski.projektksiegowy.invoice.service.InvoiceBook;
+import pl.tomaszgronski.projektksiegowy.invoice.validator.InvoiceValidator;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.logging.Logger;
+
+
+@SpringBootTest
+@RunWith(SpringRunner.class)
+
+public class InvoiceBookTestMock {
+
+    private static final Logger LOGGER = Logger.getLogger(pl.tomaszgronski.projektksiegowy.invoice.InvoiceControllerTest.class.toString());
```

Comment:
use Logger.getLogger(this.getClass().getName()) so you have the correct name even if the class name changes